### PR TITLE
[48560] Disable consents functionality for release 1

### PR DESCRIPTION
**Pull request by nc-kano**

### CODE REVIEW

**Review by fsnNC — APPROVED** [review]

File: NDB.Covid19/NDB.Covid19.Droid/Utils/NavigationHelper.cs [hunk 1]
```diff
@@ -96,7 +97,9 @@ public static Intent GetStartPageIntent(Activity parent)
 
         public static void GoToStartPageIfIsOnboarded(Activity parent)
         {
-            if (OnboardingStatusHelper.Status == OnboardingStatus.CountriesOnboardingCompleted)
+            if ((Conf.IsReleaseOne &&
```
Comment (matishadow) [inline NDB.Covid19/NDB.Covid19.Droid/Utils/NavigationHelper.cs:100]: Inner parentheses are redundant.

File: NDB.Covid19/NDB.Covid19.iOS/Utils/NavigationHelper.cs [hunk 2]
```diff
@@ -125,7 +126,9 @@ public static void GoToResultPage(UIViewController parent, bool fromOnboarding)
 
         public static void GoToResultPageIfOnboarded(UIViewController parent)
         {
-            if (OnboardingStatusHelper.Status == OnboardingStatus.CountriesOnboardingCompleted)
+            if ((Conf.IsReleaseOne &&
```
Comment (matishadow) [inline NDB.Covid19/NDB.Covid19.iOS/Utils/NavigationHelper.cs:129]: Inner parentheses are redundant.

**Review by matishadow — CHANGES_REQUESTED** [review]
I'm not too familiar with the project yet so take this comment with a grain of salt. 


The logic you introduced with `Conf.IsReleaseOne` is pretty scattered across the project.
Have you considered putting it in one class so it's all in one place? 

Plus the code is duplicated in both `NavigationHelper` classes.

File: NDB.Covid19/NDB.Covid19.iOS/Utils/NavigationHelper.cs (hunk 2 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.iOS/Utils/NavigationHelper.cs]: done

File: NDB.Covid19/NDB.Covid19.Droid/Utils/NavigationHelper.cs (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/Utils/NavigationHelper.cs]: done

**Comment by nc-kano** [comment]
@matishadow Moved all IsReleaseOne logic to ConsentsHelperClass

**Review by matishadow — APPROVED** [review]
Really good!

**Pull request merged**